### DISCUSSION TITLE
is the DFP-34x-2c2 compatible with a Fritz!Box 5530/5590 ? Could be somehow became compatible? Is there a firmware appropriate?

### DISCUSSION THREAD

**Discussion by jimpapi** [Anime4000/RTL960x#227] [Q&A]
I have a Fritz!Box 5590 with the GPON sfp from avm. But my ftth provider does not support personal routers. It comes with the nokia G-010G-Q. I have found the way to copy the necessary information from this ONT and know PLOAM password , SoftwareVersion ecc.
But is it possible to use te DFP-34x-2c2 xgpon stick after configure it the right way to simulate the Nokia ont to make it work directly to the sfp port on my Fritz!Box? 
Thanks in advance. 
I still waiting for the  DFP-34x-2c2 to be delivered so i haven't really tried to place it on my 5590 yet!

**Comment by Anime4000** [comment]
it should right away if you enter correctly, before my ISP use Nokia OLT, I need set HW Version and PLOAM Password to work.

Maybe ISP order an Nokia OLT to support Universal ONU,

Maybe you can try buy used cheap Huawei ONT and try that?

**Reply by jimpapi** [reply]
i am sorry i didn't my self clear.. The problem for me isn't how to fake the nokia ont. The problem is i want to use it in the sfp port of the fritzbox 5590 for this  DFP-34x-2c2.
Since fritzbox has GPON SoC in the router and not to the sfp, all the connection work like ploam , serial number ecc, comes from the router it self. So i don't know if DFP-34x-2c2 could work in something like bridge mode?
